app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -134,6 +134,10 @@ protected void onCreate(Bundle savedInstanceState) {
         } else {
             Log.d(TAG, "Activity was started with url " + feedUrl);
             setLoadingLayout();
+            //Remove subscribeonandroid.com from feed URL in order to subscribe to the actual feed URL
+            if(feedUrl.contains("subscribeonandroid.com")){
+                feedUrl = feedUrl.replaceFirst("((www.)?(subscribeonandroid.com/))","");
+            }
```

Comment:
Please use the Google java code style. Basically, add more space characters like in the statements below. Next to method arguments and curly braces. This is currently not checked on CI because it is too inconsistent in the code base but I would prefer new code to be consistent.